CHANGELOG.md

```diff
@@ -153,8 +153,9 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 - Fix registration and initialization of multiple extensions ([10256](https://github.com/opensearch-project/OpenSearch/pull/10256))
 - Fix Segment Replication ShardLockObtainFailedException bug during index corruption ([10370](https://github.com/opensearch-project/OpenSearch/pull/10370))
 - Fix some test methods in SimulatePipelineRequestParsingTests never run and fix test failure ([#10496](https://github.com/opensearch-project/OpenSearch/pull/10496))
+- Fix flaky query profile phase tests with concurrent search enabled ([#10547](https://github.com/opensearch-project/OpenSearch/pull/10547))
```

Comment:
Not need to put fixes to tests in the changelog.

Reply:
Thanks, @andrross. It has addressed now.